File: Base/src/main/java/picture/luck/picture/lib/app/IApp.java
package picture.luck.picture.lib.app;

import android.content.Context;

import picture.luck.picture.lib.engine.PictureSelectorEngine;

/**
 * @author：luck
 * @date：2019-12-03 15:14
 * @describe：IApp
 */
public interface IApp {
    /**
     * Application
     *
     * @return
     */
    Context getAppContext();

    /**
     * PictureSelectorEngine
     *
     * @return
     */
    PictureSelectorEngine getPictureSelectorEngine();
}

File: PictureSelector/src/main/java/com/luck/picture/lib/app/IApp.kt
package com.luck.picture.lib.app

import android.content.Context
import com.luck.picture.lib.engine.PictureSelectorEngine

/**
 * @author：luck
 * @date：2019-12-03 15:14
 * @describe：IApp
 */
interface IApp {
    /**
     * Application
     *
     * @return
     */
    val appContext: Context?

    /**
     * PictureSelectorEngine
     *
     * @return
     */
    val pictureSelectorEngine: PictureSelectorEngine?
}